src/realm/util/platform_info.hpp

```diff
@@ -36,6 +36,39 @@ inline std::string get_platform_info()
             info.machine); // Throws
 }
 
+inline std::string get_library_platform()
+{
+#if REALM_ANDROID
+    return "Android";
+#elif REALM_WINDOWS
+    return "Windows";
+#elif REALM_UWP
+    return "UWP";
+#elif REALM_IOS
+    return "iOS";
+#elif REALM_PLATFORM_APPLE
+    return "MacOS";
+#elif REALM_LINUX
+    return "Linux";
+#endif
+
+    return "unknown";
+}
+
+inline std::string get_library_cpu_arch()
+{
+#if REALM_ARCHITECTURE_ARM32
+    return "armeabi-v7a";
+#elif REALM_ARCHITECTURE_ARM64
+    return "arm64";
```

Comment:
I wonder if it would be more consistent to call this "arm64-v8a"

Checking https://docs.google.com/document/d/1jUeN71gzQoLoYxHUawzIQaDVD2xDLbtbWkmxjIMcN-U I see you followed the spec to the letter, but I guess we can still change it, if others agree.

Reply:
Both are used, depending on what platform you ask, we see both values on Kotlin Multiplatform. I don't have strong preferences either way so 🤷. I would slightly prefer `arm64` just because it is easier on the eyes and easier to write, but if anyone feels strongly about either one, I would just go with that.

Reply:
We're kind of nitpicking here, but I also don't see much value in overly verbose names. So I'd vote for `x86`, `x64`, `arm`, and `arm64`.

Reply:
I agree with Nikola about not using overly verbose names, but if I understand correctly the device architecture can be seen by users looking at logs on the backend, so maybe it should make sense in the context of the platform. `x64` is specifically a Microsoft thing, but `x86_64`, `x86`, `arm64`, and `arm` are certainly generic enough.

Reply:
I'm fine with that 👍  I just feel the current discrepancy in verbosity was odd, ie. `armeabi-v7a` vs `arm64`. You suggestion @nirinchev jives well with me, and now that we only have to change it here in realm-core it shouldn't take long.